src/settings/_box-style.scss

```diff
@@ -3,3 +3,7 @@
 
 // Border radius
 $bdrs-m: 2px !default;
+
+// Form focus
+$bxsh-input-focus: 0 0 3px 0 #60acd0 !default;
```

Comment:
Isn't this reusable by a div, for instance? if it is, this name might be confusing.

Instead of `#60acd0` please use `color-variation($c-primary, 1)`. We are building themes for all of our brands and we can't have hardcoded colors. If you wonder how I know that 60acd0 is that variation, check out this [mixin](https://github.com/SUI-Components/sui-theme/blob/master/src/utils/_colors.scss#L15) and the color section at https://app.zeplin.io/project/57c02a80c1caf50b0d2e851e/screen/598af71c138655ba087ce28c

Reply:
okay, make sense